terraform-kickstarter/prowler_build_spec.yml

```diff
@@ -0,0 +1,25 @@
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      python: 3.8
+    commands:
+      - echo "Installing Prowler and dependencies..."
+      - pip3 install detect-secrets 
+      - yum -y install jq
+      - curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+      - unzip awscliv2.zip
+      - ./aws/install
+      - git clone https://github.com/toniblyx/prowler
+      
+  build:
+    commands:
+      - echo "Running Prowler as ./prowler $PROWLER_OPTIONS"
+      - cd prowler
+      - git checkout 2.5
```

Comment:
I'd move these lines, together with the `git clone` from above to the pre-build phase instead.
At the same time, why are we forcing 2.5, what will happen when a new version appears?

Reply:
Thanks for the comment and made the changes as suggested.

Reply:
perfect!